packages/react-native-bpk-component-navigation-bar/README.md

```diff
@@ -159,12 +159,13 @@ import { BpkNavigationBarTextButtonIOS } from 'react-native-bpk-component-naviga
 
 #### Props
 
-| Property  | PropType | Required | Default Value |
-| --------- | -------- | -------- | ------------- |
-| title     | string   | true     | -             |
-| emphasize | bool     | false    | false         |
-| onPress   | func     | false    | null          |
-| disabled  | bool     | false    | false         |
+| Property  | PropType                     | Required | Default Value |
+| --------- | ---------------------------- | -------- | ------------- |
+| title     | string                       | true     | -             |
+| disabled  | bool                         | false    | false         |
+| emphasize | bool                         | false    | false         |
+| onPress   | func                         | false    | null          |
+| type      | oneOf('default', 'primary')  | false    | primary       |
```

Comment:
`'default'` should be the default otherwise this becomes a breaking change

Reply:
That's a typo - it is default. Good spot!